Multisig/UI/App/UIKit/AppSettingsViewController/AppSettingsViewController.swift

```diff
@@ -0,0 +1,226 @@
+            self.removeImportedOwnerKey()
+        }
+        return cell
+    }
+
+    #warning("TODO: show snackbar message")
+    private func removeImportedOwnerKey() {
+        let alertController = UIAlertController(
+            title: nil,
+            message: "Removing the owner key only removes it from this app. It doesn't delete any Safes from this app or from blockchain. For Safes controlled by this owner key, you will no longer be able to sign transactions in this app",
+            preferredStyle: .actionSheet)
+        let remove = UIAlertAction(title: "Remove", style: .destructive) { [unowned self] _ in
+            do {
+                try App.shared.keychainService.removeData(
+                    forKey: KeychainKey.ownerPrivateKey.rawValue)
+                AppSettings.setSigningKeyAddress(nil)
+//                App.shared.snackbar.show(message: "Owner key removed from this app")
+                Tracker.shared.setUserProperty("0", for: TrackingUserProperty.numKeysImported)
+                self.reload()
+            } catch {
+                LogService.shared.error(error.localizedDescription)
+                App.shared.snackbar.show(message: error.localizedDescription)
+            }
+        }
+        let cancel = UIAlertAction(title: "Cancel", style: .cancel, handler: nil)
+        alertController.addAction(remove)
+        alertController.addAction(cancel)
+        present(alertController, animated: true)
+    }
+
+    // MARK: - Table view delegate
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+        let item = sections[indexPath.section].items[indexPath.row]
+        switch item {
+        case Section.General.importKey(_):
+            let enterSeedVC = EnterSeedPhraseView().hostSnackbar()
+            let hostingController = UIHostingController(rootView: enterSeedVC)
+            present(hostingController, animated: true)
+        case Section.General.terms(_):
+            openInSafari(legal.termsURL)
+        case Section.General.privacyPolicy(_):
+            openInSafari(legal.privacyURL)
+        case Section.General.licenses(_):
+            openInSafari(legal.licensesURL)
+        case Section.General.getInTouch(_):
+            let getInTouchVC = GetInTouchView()
+            let hostingController = UIHostingController(rootView: getInTouchVC)
+            show(hostingController, sender: self)
+        case Section.Advanced.advanced(_):
+            let advancedVC = AdvancedAppSettings()
+            let hostingController = UIHostingController(rootView: advancedVC)
+            show(hostingController, sender: self)
+        default:
+            break
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let view = tableView.dequeueHeaderFooterView(BasicHeaderView.self)
+        view.setName("")
```

Comment:
Is there a default text that needs to be overridden? Maybe we can set the empty string in the header itself?